src/Components/Notifications/ArticleFeaturedArtistNotification.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  Flex,
+  Spacer,
+  Text,
+  Box,
+  ResponsiveBox,
+  Image,
+  Button,
+} from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import { FC } from "react"
+import { useFragment, graphql } from "react-relay"
+import { NotificationTypeLabel } from "Components/Notifications/NotificationTypeLabel"
+import { ArticleFeaturedArtistNotification_notification$key } from "__generated__/ArticleFeaturedArtistNotification_notification.graphql"
+import { extractNodes } from "Utils/extractNodes"
+import { FollowArtistButtonQueryRenderer } from "Components/FollowButton/FollowArtistButton"
+
+const MAX_WIDTH = 600
+
+interface ArticleFeaturedArtistNotificationProps {
+  notification: ArticleFeaturedArtistNotification_notification$key
+}
+
+export const ArticleFeaturedArtistNotification: FC<ArticleFeaturedArtistNotificationProps> = ({
+  notification,
+}) => {
+  const notificationData = useFragment(
+    ArticleFeaturedArtistNotificationFragment,
+    notification
+  )
+
+  const { headline, item } = notificationData
+
+  const article = item?.article
+  const artists = extractNodes(item?.artistsConnection)
+  const image = article?.thumbnailImage?.cropped
+
+  if (!article || !artists) {
+    return (
+      <Text variant="lg" m={4}>
+        Sorry, something went wrong.
+      </Text>
+    )
+  }
+
+  return (
+    <Box mx={4} my={4}>
+      <Flex width="100%" justifyContent="space-between">
+        <Flex flex={1}>
+          <Text fontWeight="bold" variant="xl">
+            {headline}
+          </Text>
+        </Flex>
+      </Flex>
+
+      <Spacer y={1} />
+
+      <NotificationTypeLabel item={notificationData} />
+
+      <Spacer y={1} />
+
+      <Flex flexDirection="row" gap={1} alignItems="center">
+        {artists.length === 1 && (
+          <FollowArtistButtonQueryRenderer
+            id={artists[0].internalID}
+            size="small"
+          />
+        )}
+
+        {artists.map(artist => {
+          return (
+            <RouterLink
+              to={artist.href}
+              key={artist.internalID}
+              textDecoration="none"
+            >
+              <Text>{artist.name}</Text>
+            </RouterLink>
+          )
+        })}
```

Comment:
**Question:** Have we discussed the case when we have more than one featured artist? I'm working on the same ticket but for Eigen - should I reuse your implementation?

Reply:
Yep, I've asked Barney if we should stick to this approach and he agreed @dariakoko

Reply:
Perfect, thank you!